Source/WebKit/WPEPlatform/wpe/wayland/WPEViewWayland.cpp

```diff
@@ -406,24 +425,40 @@ static const struct zwp_linux_dmabuf_feedback_v1_listener linuxDMABufFeedbackLis
     [](void* data, struct zwp_linux_dmabuf_feedback_v1*)
     {
         auto* view = WPE_VIEW_WAYLAND(data);
+
+        // The compositor might not have sent the formats table. In that case, try to reuse the previous
+        // one. Return early and skip emitting the signal if there is no usable formats table in the end.
+        if (!view->priv->pendingDMABufFeedback->formatTable) {
```

Comment:
```suggestion
        if (view->priv->pendingDMABufFeedback && !view->priv->pendingDMABufFeedback->formatTable) {
```

Reply:
Wouldn't it rather be “if there's no pending feedback object, return early”?

Reply:
Ah, I see you suggest the early return below 👌🏼